frontend/src/components/blocks/BlockDetails.tsx

```diff
@@ -101,9 +101,13 @@ export default ({ block }: Props) => {
                   </Term>
                 }
                 text={
-                  <BlockLink blockHash={block.prevHash}>
-                    {block.prevHash}
-                  </BlockLink>
+                  block.prevHash === "Genesis" ? (
```

Comment:
After addressing the first comment, use `11111111111111111111111111111111` here.